tests/static/__init__.py

```diff
@@ -0,0 +1,47 @@
+import json
```

Comment:
To do that, I guess we should wait till https://github.com/openwisp/netengine/pull/57 is merged, so that we can create a `tests/utils.py` because currently `tests/utils/__init__.py` exists with a precedence over that file, and it's going to be removed anyway :smile:

Reply:
I overlooked that `tests/utils/` already exist here. My main concern was removing this logic from `tests/static/__init__.py`